flask_api/expiry.json

```diff
@@ -0,0 +1 @@
+{"Testup": {"expiry": "20240716", "count": 1, "timestamp": "20240703"}, "Testup1": {"expiry": "20240717", "count": 1, "timestamp": "20240703"}, "Testup2": {"expiry": "20240715", "count": 1, "timestamp": "20240703"}}
```

Comment:
Might change the format of this json so its easier to work with on the FE side

Instead of having the food name be the key at the top level, embed it inside as `"food": "Ketchup"`

Reply:
The issue is that the top level key is always going to be dynamic so its more difficult to parse out